content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+**trust bundles** must be federated between them. See the section
+on [Multiple Trust Domains](#trust-between-meshes) for an overview.
+
+## Control plane models
+
+The Istio [control plane](https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-control-plane)
+configures the mesh or a subset of the mesh to manage the communication between
+workload instances. A single deployment of the control plane consists of several
+redundant processes that share a common view of the mesh and provide workload
+instances with identical configuration.
+
+Depending on your overall system architecture and cloud provider, the following
+Istio control plane models are possible:
+
+- [single control plane](#single-control-plane)
+- [multiple control planes](#multiple-control-planes)
+- [managed control planes](#managed-control-plane)
+
+### Single control plane
+
+In the simplest case, you can run your mesh using a single control plane
+deployment. The single control plane model is typically used for [single
+cluster](#single-cluster) meshes.
+
+If you run your mesh on a single cluster, a single control plane deployment
+suffices. Each workload instance in the cluster connects to a control plane
+instance to get its configuration.
+
+{{< image width="50%"
+    link="./single-cluster.svg"
+    alt="A service mesh with a single control plane"
+    title="Single control plane"
+    caption="A service mesh with a single control plane"
+    >}}
+
+### Multiple control planes
+
+You can deploy multiple control planes across the mesh; such a configuration is
+known as a **multi-control plane**.
+
+Multiple control planes afford improved performance and availability for meshes
+that span [multiple clusters](#multiple-clusters) across multiple zones and
+regions.
+
+Istio supports control plane deployments at various levels of availability:
+
+- Control plane per region: *lowest availability*
+- Control plane per zone
+- Control plane per cluster *highest availability*
+
+The configuration of each control plane could be identical or different
+depending on the [network](#network-models) and [cluster](#cluster-models)
+models of the system.
+
+{{< image width="50%"
+    link="./multi-control.svg"
+    alt="A service mesh with multiple control planes"
+    title="Multiple control planes"
+    caption="A service mesh with multiple control planes"
+    >}}
+
+### Managed control plane
+
+Many cloud providers offer a control plane, which they manage on behalf of their
+customer.
+
+Typically, managed control planes guarantee some level of performance and
+availability, which you can assume when building your system. Managed control
+planes affords the following benefits:
+
+- Greatly reduce the complexity of user deployments
+- Greatly reduce system administration costs
+- Effectively eliminate the usage costs associated with the control plane:
+    - CPU
+    - Memory
+    - Network
+
+{{< image width="50%"
+    link="./managed-control.svg"
```

Comment:
Should we have the managed control plane outside the mesh rather than inside?

Reply:
Oof yes ... I didn't notice that when I reviewed previously.

@rcaballeromx maybe we can move the managed control plane to the top or bottom of the diagram to keep it outside the mesh?

Reply:
I think it's reasonable to show it inside the mesh. Even though the control plane components are not really "in the mesh", even in the other control plane diagrams, they're usually running in the same cluster as the mesh services, so you can sort of think of it as part of the mesh for those clusters. So, in the same way, there's a conceptual managed control plane that is dedicated to each mesh, so the diagram seems OK to me.

Reply:
@frankbu The point of this section is to abstract how the managed control plane is implemented ... it's not a concern of the user.

A managed control plane is inherently *not* part of the mesh and it should be represented as such in the diagram. The cloud provider will likely deploy an instance of their managed control plane per user mesh ... but again, it's an implementation detail. I think if we consistently picture the managed control plane outside of the mesh we avoid confusing the reader.

Reply:
I still think that even in the non-managed control plane case the control plane is not part of the mesh, yet it is shown inside the mesh in those diagrams. So for consistency, I think it would be less confusing to leave it inside the mesh box in all the control plane diagrams. That said, I don't have a strong objection to changing it, so whatever you choose is fine.

Reply:
For non-managed control planes, it gets tricky ... particularly in the case of multi-cluster, since you have to show the control plane in each cluster.

For non-managed I think it's generally helpful to picture the control plane inside the mesh to show that it's dedicated to that mesh. This is especially helpful in multi-mesh scenarios. 

Managed is a different animal altogether as the topology is purposely left unknown, so outside of the mesh is the only thing that makes sense.

Reply:
Fine. But I think you're getting to the heart of the issue with control plane configuration: physical vs. logical, shared vs. dedicated, replicated and synchronized vs. federated. I think we need to explain these things.

Reply:
Managed control plane doesn't necessarily mean that the control plane is out of the mesh right? Cloud providers can manage the Istio control plane (installation, updates, health), but still have it residing in the cluster. This also means that usage costs concerns are not eliminated.

Reply:
Of course, cloud providers implement it however they choose. The physical location of the managed control plane shouldn't generally be a concern for the end-user.

We probably shouldn't make too many assumptions about cloud provider pricing models. Maybe we can restate it this way:

```
Eliminate many of the usage costs associated with the control plane
```